library/src/main/java/com/google/maps/android/data/Renderer.java

```diff
@@ -140,6 +149,29 @@ public Renderer(GoogleMap map, HashMap<? extends Feature, Object> features) {
         mContainerFeatures = null;
     }
 
+    /**
+     * Fragment for retaining the bitmap cache between configuration changes.
+     */
+    public static class RetainFragment extends Fragment {
+        private static final String TAG = RetainFragment.class.getName();
```

Comment:
So this results in a cache in this fragment which is used globally for all activities (i.e., single cache per process). This seems reasonable to me, because if it were scoped per activity you'd likely have duplicates across the caches.